egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -0,0 +1,312 @@
+                        "(prior to merging egs). ")
+    parser.add_argument("--num-jobs", type=int, default=20,
+                        help="This can be used for better randomness in distributing "
+                        "languages across archives, where egs.job.archive.scp generated "
+                        "randomly and examples are combined "
+                        "across all jobs as eg.archive.scp.")
+    parser.add_argument("--random-lang", type=str, action=common_lib.StrToBoolAction,
+                        help="If true, the lang-id in ranges.* selected"
+                        " w.r.t frequency distribution of remaining examples in each language,"
+                        " otherwise it is selected sequentially.",
+                      default=True, choices = ["false", "true"])
+    parser.add_argument("--max-archives", type=int, default=1000,
+                        help="max number of archives used to generate egs.*.scp")
+    parser.add_argument("--seed", type=int, default=1,
+                        help="Seed for random number generator")
+
+    parser.add_argument("--minibatch-size", type=int, default=512,
+                        help="The minibatch size used to generate scp files per job. "
+                             "It should be multiple of actual minibatch size.")
+
+    parser.add_argument("--prefix", type=str, default="",
+                        help="Adds a prefix to the range files. This is used to "
+                        "distinguish between the train and diagnostic files.")
+
+    parser.add_argument("--lang2weight", type=str,
+                        help="lang2weight file contains the weight per language to "
+                           "scale output posterior for that language.(format is: "
+                           "<lang-id> <weight>)")
+    parser.add_argument("--reporting.email", dest="email",
+                        type=str, default=None,
+                        action=common_lib.NullstrToNoneAction,
+                        help=""" Email-id to report about the progress
+                        of the experiment.  NOTE: It assumes the
+                        machine on which the script is being run can
+                        send emails from command line via. mail
+                        program. The Kaldi mailing list will not
+                        support this feature.  It might require local
+                        expertise to setup. """)
+# now the positional arguments
+    parser.add_argument("num_langs", type=int,
+                        help="num of languages used in multilingual training setup.")
+    parser.add_argument("egs_scp_lists", type=str,
+                        help="list of egs.scp files per input language."
+                           "e.g. exp/lang1/egs/egs.scp exp/lang2/egs/egs.scp")
+
+    parser.add_argument("egs_dir",
+                        help="Name of egs directory e.g. exp/multilingual_a/egs")
+
+
+    print(' '.join(sys.argv))
```

Comment:
This comment was not addressed.  The majority of Vimal's comments seem not to have been addressed.  Do you have changes you have not yet pushed?